### PR TITLE
update: okta instructions

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/howto/saml/add-identity-providers.md [hunk 1]
```diff
@@ -9,19 +10,12 @@ provider (IdP).
...
 To set up single sign-on through an IdP for your organization:
 
-1. Add the identity provider in the [Aiven
-    Console](https://console.aiven.io/) .
-1. Configure SAML on your IdP.
-1. Finalize the setup in the Aiven Console using information from your
-    IdP.
-1. Link your users to the identity provider.
-
 ## Step 1. Add the IdP in the Aiven Console {#add-idp-aiven-console}
```
Comment (harshini-rangaswamy) [inline docs/platform/howto/saml/add-identity-providers.md:14]: Should we include a link to the Aiven Console here?

File: docs/platform/howto/saml/add-identity-providers.md [hunk 2]
```diff
@@ -9,19 +10,12 @@ provider (IdP).
...
 
 1. In the organization, click **Admin**.
 1. Click **Identity providers**.
 1. Click **Add identity provider**.
-1. Select an IdP and enter a name.
+1. Select SAML and enter a name.
 1. Select a [verified domain](/docs/platform/howto/manage-domains) to link
     this IdP to. Users see linked IdPs on the login page.
```
Comment (harshini-rangaswamy) [inline docs/platform/howto/saml/add-identity-providers.md:20]: Indentation is off

File: docs/platform/howto/saml/add-okta-idp.md [hunk 3]
```diff
@@ -1,70 +1,57 @@
...
 
-    :::note
-    The `Default RelayState` is the homepage of the Aiven Console and is
-    necessary for IdP-initiated login to function correctly.
-    :::
+## Configure SAML on Okta {#configure-saml-okta}
 
-1.  Add an entry to the **Attribute Statements** with the following:
+1.  In the [Okta administrator console](https://login.okta.com/), go to
+     **Applications** > **Applications**.
```
Comment (harshini-rangaswamy) [inline docs/platform/howto/saml/add-okta-idp.md:24]: Indentation

File: docs/platform/howto/saml/add-identity-providers.md (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/saml/add-identity-providers.md:14]: I have no made a habit of adding this. Maybe something to discuss with the team.

**Review by rsalevsky — APPROVED** [review]

**Pull request merged**